infra-templates/ecr/1/rancher-compose.yml

```diff
@@ -32,5 +32,31 @@
       default: "INFO"
       required: true
       type: "string"
+    - variable: "registry_in_which_environment"
+      type: enum
+      label: "Registry Environment"
+      description: "Which environemnt is the AWS registry located?"
+      default: current
+      options:
+        - current
+        - other
+    - variable: "environment_api_endpoint"
+      label: "URL (Environment API Endpoint) of Registry to be Updated"
+      description: "URL for where the registry is located. This is the endpoint for an Environment API Key, which can be found under the Advanced Optios under API."
```

Comment:
which can be found under API -> Keys

Reply:
I think this is better
`URL for where the registry is located. This is the endpoint, which can be found under the Advanced Options under API, for an Environment API Key.`